keyboards/nopunin10did/kastenwagen/info.json

```
@@ -0,0 +1,134 @@
+                {"x":2   ,  "y":0,    "label":"W"},
+                {"x":3   ,  "y":0,    "label":"E"},
+                {"x":4   ,  "y":0,    "label":"R"},
+                {"x":5   ,  "y":0,    "label":"T"},
+                {"x":6   ,  "y":0,    "label":"Y"},
+                {"x":7   ,  "y":0,    "label":"U"},
+                {"x":8   ,  "y":0,    "label":"I"},
+                {"x":9   ,  "y":0,    "label":"O"},
+                {"x":10  ,  "y":0,    "label":"P"},
+                {"x":11  ,  "y":0,    "label":"L Bracket"},
+                {"x":12  ,  "y":0,    "label":"Backspace", "w":1.75},
+                {"x":14.25, "y":0,    "label":"Delete"},
+                {"x":0   ,  "y":1,    "label":"Tab", "w":1.25},
+                {"x":1.25,  "y":1,    "label":"A"},
+                {"x":2.25,  "y":1,    "label":"S"},
+                {"x":3.25,  "y":1,    "label":"D"},
+                {"x":4.25,  "y":1,    "label":"F"},
+                {"x":5.25,  "y":1,    "label":"G"},
+                {"x":6.25,  "y":1,    "label":"H"},
+                {"x":7.25,  "y":1,    "label":"J"},
+                {"x":8.25,  "y":1,    "label":"K"},
+                {"x":9.25,  "y":1,    "label":"L"},
+                {"x":10.25, "y":1,    "label":"Semicolon"},
+                {"x":11.25, "y":1,    "label":"Quote"},
+                {"x":12.25, "y":1,    "label":"Enter", "w":1.5},
+                {"x":14.25, "y":1,    "label":"Pg Up"},
+                {"x":0   ,  "y":2,    "label":"Left Shift", "w":1.75},
+                {"x":1.75,  "y":2,    "label":"Z"},
+                {"x":2.75,  "y":2,    "label":"X"},
+                {"x":3.75,  "y":2,    "label":"C"},
+                {"x":4.75,  "y":2,    "label":"V"},
+                {"x":5.75,  "y":2,    "label":"B"},
+                {"x":6.75,  "y":2,    "label":"N"},
+                {"x":7.75,  "y":2,    "label":"M"},
+                {"x":8.75,  "y":2,    "label":"Comma"},
+                {"x":9.75,  "y":2,    "label":"Period"},
+                {"x":10.75, "y":2,    "label":"Slash"},
+                {"x":11.75, "y":2,    "label":"Right Shift", "w":1.25},
+                {"x":13.125,"y":2.125,"label":"Up"},
+                {"x":14.25, "y":2,    "label":"Pg Dn"},
+                {"x":0    , "y":3,    "label":"Left Ctrl", "w":1.25},
+                {"x":1.25 , "y":3,    "label":"Left Alt", "w":1.25},
+                {"x":2.5  , "y":3,    "label":"MO(2)"},
+                {"x":3.5  , "y":3,    "label":"Space", "w":2.75},
+                {"x":6.25 , "y":3,    "label":"Space", "w":2.25},
+                {"x":8.5  , "y":3,    "label":"MO(3)"},
+                {"x":9.5  , "y":3,    "label":"R Win / Alt Gr", "w":1.25},
+                {"x":10.75, "y":3,    "label":"MO(1)", "w":1.25},
+                {"x":12.125,"y":3.125,"label":"Left"},
+                {"x":13.125,"y":3.125,"label":"Down"},
+                {"x":14.125,"y":3.125,"label":"Right"},
```

Fixes a syntax error that prevents the file from working.

```suggestion
                {"x":14.125,"y":3.125,"label":"Right"}
```